src/gtest/test_blockdownload.cpp

```diff
@@ -0,0 +1,95 @@
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include "main.h"
+
+using namespace zen;
+
+void makeTemporaryMainChain(int trunk_size);
```

Comment:
This seems to be a leftover from the previous implementation, we should probably remove it?